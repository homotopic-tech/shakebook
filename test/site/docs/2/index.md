---
title: The Second Subsection
description: The Second Subsection
---

Lorem ipsum
