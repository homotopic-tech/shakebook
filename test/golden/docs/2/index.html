<html>
  <head>
    <title>The Second Subsection</title>
  </head>
  <body>
    <h1>This post page is called The Second Subsection</h1>
    <p>It was generated from the source docs/2/index.md</p>
    <p>For the url /docs/2/index.html</p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the subsections:
      <ul>
        <li>
          <a href="/docs/2/champ.html">Hi Iâ€™m champ</a>
          <p>Woof</p>
        </li>
      </ul>
    </p>
  </body>
</html>
